includes/derivatives.inc

```diff
@@ -126,9 +126,6 @@ function islandora_ocr_derive_datastreams(AbstractDatastream $datastream, array
   $ret = $ret && islandora_ocr_update_datastream($datastream->parent, $derivatives['OCR'], 'OCR');
   $ret = $ret && $derivatives['HOCR'];
   $ret = $ret && islandora_ocr_update_datastream($datastream->parent, $derivatives['HOCR'], 'HOCR');
-
-  $files = array_filter(array($source_file, $ocr_file, $hocr_file));
```

Comment:
Hi @ajstanley. Cool. I see why removing this
```PHP
 array_filter(array($source_file, $ocr_file, $hocr_file));
````
makes sense (none of those were defined at all!). 
What I don't see is how you manage to remove (unlink) the [files created indirectly via islandora_ocr_generate_derivatives()](https://github.com/ajstanley/islandora_ocr/blob/d57b4915055905075db33c2eee4da50c09c3cb09/includes/derivatives.inc#L184-L185) when calling [islandora_ocr_create_ocr](https://github.com/ajstanley/islandora_ocr/blob/d57b4915055905075db33c2eee4da50c09c3cb09/includes/derivatives.inc#L20-L38)  and [islandora_ocr_create_hocr](https://github.com/ajstanley/islandora_ocr/blob/d57b4915055905075db33c2eee4da50c09c3cb09/includes/derivatives.inc#L52-L93)

Should that be managed here, after returning or is that already managed somewhere else?